static/css/main.scss

```diff
@@ -0,0 +1,440 @@
+$white: #ffffff;
+$newsprint-tint1: #fafaf7;
+$newsprint: #efefea;
+$newsprint-shade1: #e6e8df;
+$newsprint-shade2: #deded8;
+$newsprint-shade3: #c8cac3;
+$maroon: #860e00;
+$maroon-shade1: #390b00;
+$black: #000000;
+$gray: #cccccc;
+$gray-shade1: #dddddd;
+$highlight: #ff0000;
+
+
+body {
+  background: $newsprint url(../images/full_backdrop.png) no-repeat center -170px;
+}
+
+// Header / Title
+
+.header {
+  padding: 1.5em 0;
+
+  h1 {
+    font-family: 'Oranienbaum', serif;
+    margin: 0;
+    padding: .2em;
+    text-align: center;
+  } 
+}
+
+@media (min-width: 768px) {
+
+  h1 {
+    .nebraska {
+      margin-left: .9em;
+    }
+    .newspapers {
+      padding-left: 1.6em;
+    }
+  }
+
+  h1.title {
+    font-size: 4em;
+    font-variant: small-caps;
+    font-weight: 700;
+    line-height: .95em;
+  }
+}
+
+@media (min-width: 992px) {
+  h1.title {
+    font-size: 6em;
+  }
+}
+
+// Navigation
+
+.navbar {
+  background-color: rgba($white, 0.7);
+  border-left: none;
+  border-right: none;
+  border: double $black 3px;
+  font-family: 'Oranienbaum';
+  font-size: 1.2em;
+  margin: 0;
+    a {
+    text-decoration: none;
+  }
+}
+
+.navbar-right {
+  font-size: .9em;
+}
+
+.navbar-second {
+  border: solid 3px $black;
+  border-left: none;
+  border-right: none;
+  border-top: none;
+}
+
+/* To center the search */
+
+@media (min-width: 768px) {
+
+  .navbar-second .container {
+    text-align: center;
+  }
+
+  .navbar-center {
+    display: inline-block;
+    margin-bottom: -7px;
+  }
+}
+
+/* calendar */
+
+.year-calendar {
+  font-size: .9em;
+
+  h3 {
+    background-color: $newsprint;
+    font-size: 1.3em;
+    margin-bottom: 0;
+    padding: .3em;
+  }
+
+  .table {
+    background-color: $white;
+    margin: 0;
+    padding: 0;
+  }
+
+  .table td {
+    padding: 4px;
+    width: 2.2em;
+  }
+
+  .table td.single {
+    background-color: $newsprint-shade2;
+  }
+
+  table.month td.multiple {
+    background-color: $newsprint-shade3;
+  }
+
+  .btn-group a {
+    color: $maroon;
+    text-decoration: none;
+  }
+}
+
+.calendar_month {
+  background-color: $white;
+  display: inline-block;
+  margin: 0 10px 0 0px;
+  text-align: center;
+  vertical-align: top;
+}
+
+.issues_single, .issues_multiple {
+  border: 1px solid $gray-shade1;
+  padding: 5px;
+}
+
+.issues_single {
+  background-color: $newsprint-shade1;
+}
+
+.issues_multiple {
+  background-color: $newsprint-shade3;
+}
+
+// Newspapers - All Titles
+
+table.tablesorter tbody tr.odd td {
```

Comment:
leaving all these because when I was messing with css, some of these did matter for specificity and I don't remember which. (I know for a fact h1.title did)